### PR TITLE
Ss.bing map issue for standard deployment

### PR DESCRIPTION
**Pull request by swetha571**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Standard deployment Bingmap keys. Even if the BingMapKey is empty sting, we should still show the StaticBingMap.

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/deviceMap/deviceMap.js [hunk 1]
```diff
@@ -167,6 +167,18 @@ class DeviceMap extends Component {
     });
   }
 
+  renderStaticMap() {
+      return <div className="map-container">
+        <div className="static-map-modal">
+          <span className="static-text">
+            {lang.MAP_MODAL_CONTENT}
+            <a className="static-link" rel="noopener noreferrer" href="https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-faq" target='_blank'>{lang.MAP_MODAL_FAQS}</a>
```
Comment (dluc) [inline src/components/deviceMap/deviceMap.js]: please remove the "en-us" from the URL

File: src/components/deviceMap/deviceMap.js [hunk 2]
```diff
@@ -181,15 +193,8 @@ class DeviceMap extends Component {
...
-            ? <div className="map-container">
-                <div className="static-map-modal">
-                  <span className="static-text">
-                    {lang.MAP_MODAL_CONTENT}
-                    <a className="static-link" rel="noopener noreferrer" href="https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-faq" target='_blank'>{lang.MAP_MODAL_FAQS}</a>
-                  </span>
-                </div>
-                <img src={StaticMapWithOutDevices} alt="StaticMap" className="static-bing-map"/></div>
-            : <div id="deviceMap" className="dashboard_device_map" />) : null }
+            ? this.renderStaticMap()
```
Comment (dluc) [inline src/components/deviceMap/deviceMap.js]: could you simplify the condition? something like
```
BingMapKey && BingMapKey !== config.STATUS_CODES.STATIC ? ...bingmap... : ...static map...
```

File: src/components/deviceMap/deviceMap.js (hunk 1 above)
Comment (swetha571) [reply src/components/deviceMap/deviceMap.js]: Just for making sure. In comps they have given me this " https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-faq". Even if i eliminate the "en-us" it s still redirecting to same link with en-us appending. Is that the actula behaviour??

Comment (swetha571) [reply src/components/deviceMap/deviceMap.js]: I am on it.

**Pull request merged**